doc/model.rst

```diff
@@ -0,0 +1,206 @@
+Model
+=====
+
+.. note::
+
+    This is mildly converted from PlasCom2 for now, sourced from `here
+    <https://bitbucket.org/xpacc-dev/plascom2/src/GoldenCopyCandidate/doc/Theory.dox>`__.
+    Do not consider this information authoritative while this notice is still
+    present.
+
+In fluid domains, the code solves the compressible Navier-Stokes equations
+in curvilinear coordinates.  The basic equations, in a Cartesian coordinate space, for the conserved mass
+density :math:`\rho`, momentum density :math:`\rho u_i`, and total energy
+density :math:`\rho E` are, in index form with summation convention are given as
+
+.. math::
+    \frac{\partial \rho}{\partial t} + \frac{\partial }{\partial x_j} \rho u_j &= S_\rho \\
+    \frac{\partial \rho u_i}{\partial t} + \frac{\partial}{\partial x_j}\left(\rho u_i u_j + p\delta_{ij} - \tau_{ij}\right) &= S_{\rho u_i} \\
+    \frac{\partial \rho E}{\partial t} + \frac{\partial}{\partial x_j}\left(\left\{\rho E + p\right\}u_j + q_j - u_i \tau_{ij}\right) &= S_{\rho E},
+
+where :math:`p` is the thermodynamic pressure, :math:`\tau_{ij}` is the
+viscous stress tensor, and :math:`q_i` is the heat flux in the :math:`i`th
+direction. :math:`S_\rho`, :math:`S_{\rho u_i}`, and :math:`S_{\rho E}` are are mass, momentum, and energy density source terms.  These equations can be written in the compact form
+
+.. math::
+
+    \frac{\partial Q}{\partial t} + \frac{\partial \vec{F}_j}{\partial x_j} = S,
+
+where :math:`Q = [\rho\,\rho \vec{u}\,\rho E]^T` is the vector of conserved
+variables, :math:`\vec{F} = \vec{F}^I - \vec{F}^V` is the flux vector account
+for both visicd and inviscid terms, and :math:`S` is the source term vector.
+
+Viscous stress constitutive relation
+------------------------------------
+
+The viscous stress tensor is defined as
+
+.. math::
+    \tau_{ij} = \mu \left(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i}\right) + \lambda \frac{\partial u_k}{\partial x_k}\delta_{ij}
+
+where :math:`\mu` and :math:`\lambda` are the first and second coefficients
+of viscosity, respectively; both may be a function of temperature.  Note
+that Stokes' hypothesis :math:`(\lambda = -\frac{2}{3}\mu)` is not
+automatically enforced and that :math:`\lambda` is related to bulk
+viscosity :math:`\mu_B` as :math:`\lambda = \mu_B - (2/3)\mu`.
+
+.. _heat-flux-constitutive:
+
+Heat flux constitutive relation
+-------------------------------
+
+The heat flux vector is defined as
+
+\f[
+q_i = - \kappa \frac{\partial T}{\partial x_i}
+\f]
```

Comment:
What is `\f]`?  Is there some math/rst documentation to point to in a readme?

Reply:
Doxygen leftovers, fixed in e0cf898.